main.js

```diff
@@ -91,6 +94,9 @@ function displayMessage() {
       message.classList.remove('hidden');
     }, 2000);
   }
+  setTimeout(function() {
+    clearMsgButton.classList.remove('hidden');
+  }, 2000);
```

Comment:
Are you using `setTimeout` here (and other spots) to time your animations to imitate loading? If so, that's very clever! Not a typical usage of `setTimeout`, but it works to hit that project requirement for sure 😄